Formula/tcl-tk.rb

```diff
@@ -136,6 +136,9 @@ def install
       system "make", "install"
     end
 
+    # Add symlink to common directory name for CMake to find headers
+    include.install_symlink "tcl-tk" => "tcl#{version.major_minor}"
```

Comment:
I wonder if it makes more sense to install the `tcl` headers here, and the `tk` headers into `include/tk#{version.major_minor}`.

Reply:
Didn't really look into it. Wasn't sure on if any build systems would have trouble with separate directories. I had just copied Debian's choice https://packages.debian.org/sid/amd64/tk8.6-dev/filelist 

I do see FreeBSD chose a separate `tk8.6` directory https://github.com/freebsd/freebsd-ports/blob/main/x11-toolkits/tk86/Makefile#L39. Most repositories use default directory structure without subdirectory.

Reply:
Ok, all in the same directory makes sense then. Do we also want to move the `itk` headers?
```
❯ ls -la $(brew --prefix tcl-tk)/include
.r--r--r-- 3.4k carlocab 22 Nov  2022 itk.h
.r--r--r--  10k carlocab 22 Nov  2022 itkInt.h
drwxr-xr-x    - carlocab 22 Nov  2022 tcl-tk
```
We could also probably just stick to a single `include/tcl8.6` instead of also having `include/tcl-tk`.